dds.sh

```diff
@@ -14,6 +14,8 @@ case $ARCHITECTURE in
     [[ ! $BOOST_ROOT ]] && BOOST_ROOT=`brew --prefix boost` ;;
 esac
 
+export DDS_LD_LIBRARY_PATH="${GCC_TOOLCHAIN_ROOT}/lib64"
```

Comment:
You should do this only if GCC_TOOLCHAIN_ROOT is defined (e.g. not the case when GCC is not there, for example on mac. Can you please protect it with an if?

Reply:
alright, fixed it.